modules/core/misc/java/test/MatTest.java

```diff
@@ -455,6 +455,20 @@ public void testGetIntIntShortArray() {
         bytesNum = sm.get(1, 1, buff11);
         assertEquals(4, bytesNum);
         assertTrue(Arrays.equals(new short[] {340, 341, 0, 0}, buff11));
+
+        Mat m2 = new Mat(new int[]{ 5, 5, 5 }, CvType.CV_16S);
```

Comment:
> 5, 5, 5

different numbers are preferable (to ensure that we don't mess with "size[]" array order in implementation):
```
5, 6, 8
```